addons/pos_adyen/static/src/app/pos_bus.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module */
+
+import { patch } from "@web/core/utils/patch";
+import { PosBus } from "@point_of_sale/app/bus/pos_bus_service";
+
+patch(PosBus.prototype, {
+    // Override
+    dispatch(message) {
+        super.dispatch(...arguments);
+
+        if (message.type === "ADYEN_LATEST_RESPONSE") {
+            this.pos
+                .getPendingPaymentLine("adyen")
+                .payment_method.payment_terminal.handleAdyenStatusResponse();
```

Comment:
used the config id as payload. if the config id doesn't match, don't handle it.
The idea is to make sure that the notification is only handled by the intended order and payment.

Reply:
You might need to pass more info in the payload to make sure the notification is handled properly by the correct order-payment combination.